### PR TITLE
driver.scroll action; scroll perf test for Stocks

### PR DESCRIPTION
**Pull request by yjbanov**

### CODE REVIEW

File: examples/stocks/test_driver/scroll_perf.dart [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_driver/driver_extension.dart';
+import 'package:flutter_driver/src/error.dart';
+import 'package:stocks/main.dart' as app;
+
+void main() {
```
Comment (abarth) [inline examples/stocks/test_driver/scroll_perf.dart]: Is this where we want to put this file?  I wonder if we should have it be in `examples/stocks/test/driver/scroll_perf.dart`.  We've been trying to have Flutter apps follow the Dart conventions for packages.

**Comment by abarth** [comment]
LGTM

File: examples/stocks/test_driver/scroll_perf.dart (hunk 1 above)
Comment (yjbanov) [inline examples/stocks/test_driver/scroll_perf.dart]: Eventually we'll move all driver tests under `test`, but today we are unable to tell them apart from unit-tests. For example, `flutter test` and `pub run test` will attempt to run a driver test, and will fail. Once we gain ability to use `@TestOn` we should be able to put all tests under `test` and run `pub run test -p flutter_driver` and have `flutter test` recognize them out-of-the-box.

Comment (abarth) [inline examples/stocks/test_driver/scroll_perf.dart]: Currently we key off the `_test.dart` suffix, so putting this `_perf.dart` file in `test` should be fine.  We should have TestOn support pretty soon.  The test package just published a release that will let us integrate in a less hacky way.

Anyway, I didn't mean to bikeshed your patch.  We can always move the file around.  :)

Comment (yjbanov) [inline examples/stocks/test_driver/scroll_perf.dart]: Yeah, there are other places where I already use `test_driver`. Also, not all the tests are performance tests. Some are simply integration tests. We could use a more generic prefix, perhaps `_e2e.dart`. The issue though, is that `pub run test` has no way to specify a custom prefix. I'll start an email thread to discuss. Should be fixable. We just need to pick something.

Comment (abarth) [inline examples/stocks/test_driver/scroll_perf.dart]: This issue doesn't need to block this patch.  LGTM

**Pull request merged**